a_locs_poly_setup.R

```diff
@@ -71,7 +72,7 @@ a_targets_list <- list(
   # where needed 
   tar_target(
     name = a_make_NW_CLP_polygons,
-    command = combine_and_simplify_sfs(a_CLP_polygons, "CLP", a_NW_polygons, "NW", "CLP_NW_polygons"),
+    command = combine_and_simplify_sfs(a_CLP_polygons, "CLP", a_NW_polygons, "NW", "CLP_NW_polygons", TRUE),
```

Comment:
Not sure what the rule for this is and is just a thought (more for ROSS as a whole)... but in calling functions, should we start having the arguments explicitly listed and linked for better understandability? (e.g. `sf_1 = a_CLP_polygons, data_group_1 = "CLP",  sf_2 = a_NW_polygons, data_group_2 = "NW", filename = "CLP_NW_polygons", simplify = TRUE`)

Reply:
yes yes yes. I totally agree with this and think it is super useful.